### PR TITLE
[nghttp2]: update to v1.49.0

### PR DESCRIPTION
**Pull request by alagoutte**

### CODE REVIEW

**Review by FrankXie05 — DISMISSED** [review]

**Comment by alagoutte** [comment]
> You have modified or added at least one vcpkg.json where you should check the `license` field.

Done !

File: ports/nghttp2/vcpkg.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
   "name": "nghttp2",
-  "version": "1.46.0",
+  "version": "1.48.0",
```
Comment (JavierMatosD) [inline ports/nghttp2/vcpkg.json:3]: It looks like the latest version of this port is1.49.0. Why are we not updating to the latest?

**Review by JavierMatosD — CHANGES_REQUESTED** [review]

**Comment by Thomas1664** [comment]
The git hash in the version database is wrong. Please run `vcpkg x-add-version nghttp2 --overwrite-version` to fix this.

File: ports/nghttp2/vcpkg.json (hunk 1 above)
Comment (alagoutte) [reply ports/nghttp2/vcpkg.json:3]: 1.49.0 is a new release from today... i will look for update after

Comment (alagoutte) [reply ports/nghttp2/vcpkg.json]: updated to 1.49.0

File: versions/n-/nghttp2.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+    {
+      "git-tree": "f2c6faba492d188ec0f72078720084f6a4dc00c3",
+      "version": "1.49.0",
+      "port-version": 0
+    },
+    {
+      "git-tree": "f9f789c8f3065244d005a567c9054508e843ebbb",
+      "version": "1.48.0",
+      "port-version": 0
+    },
```
Comment (FrankXie05) [inline versions/n-/nghttp2.json:12]:
```suggestion
```

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

**Review by FrankXie05 — APPROVED** [review]

**Comment by JavierMatosD** [comment]
Thank you!

**Review by JavierMatosD — APPROVED** [review]

**Pull request merged**